packages/cli-types/src/index.ts

```diff
@@ -106,6 +106,8 @@ export interface Config {
       hooks: {
         prelink?: string;
         postlink?: string;
+        preulink?: string;
```

Comment:
I think this was a typo in the command itself, should probably be `preunlink`

Reply:
Yup. It seems like this is the case.

Reply:
yeah my mistake. Will fix it soon !

Reply:
done